core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/install/ClusterManagerRefreshParcelClusterHandler.java

```diff
@@ -0,0 +1,49 @@
+package com.sequenceiq.cloudbreak.reactor.handler.cluster.install;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.common.event.Selectable;
+import com.sequenceiq.cloudbreak.core.cluster.ClusterBuilderService;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.install.ClusterManagerRefreshParcelFailed;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.install.ClusterManagerRefreshParcelRequest;
+import com.sequenceiq.cloudbreak.reactor.api.event.cluster.install.ClusterManagerRefreshParcelSuccess;
+import com.sequenceiq.flow.event.EventSelectorUtil;
+import com.sequenceiq.flow.reactor.api.handler.EventHandler;
+
+import reactor.bus.Event;
+import reactor.bus.EventBus;
+
+@Component
+public class ClusterManagerRefreshParcelClusterHandler implements EventHandler<ClusterManagerRefreshParcelRequest> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClusterManagerRefreshParcelClusterHandler.class);
+
+    @Inject
```

Comment:
oh c'mon, you get it 😄  please dont' make me to comment all the **@Inject**'s 😄

Reply:
Maybe we shouldn't mix the injections

Reply:
* by force, if someone creates a new class and makes it with constructor injection, ok, but forcing to mix the methods, is not a good way

Reply:
who said that I would like to make anybody to mix it? :)
in new classes, there is no question about this. use injection in a preferred way.
IMO one should use injection in one and only one way in any class. but as you know my opinion is that field injection is less preferred hence if we introduce new fields and one also prefers to use constructor injection then all the other fields should be moved into the constructor. this is not the case here since this is a completely new class.

but let me note that this comment is just a comment, not a change request :)